doc/kci_rootfs.md

```diff
@@ -0,0 +1,157 @@
+### How to build a rootfs image using kci_rootfs
+
+You will be using `kernelci/staging-debos` docker image for this purpose.
+
+1. Pull the docker image `docker pull kernelci/staging-debos`
+
+2. Clone the kernelci-core repo.
+
+    ```
+    git clone https://github.com/kernelci/kernelci-core.git
+    ```
+3. Start the docker and get into it.
+
+   ```
+   sudo docker run -itd -v $(pwd)/kernelci-core:/kernelci-core --device /dev/kvm --privileged kernelci/staging-debos
+   sudo docker exec -it <container_id> bash
+   cd /kernelci-core/
+   ```
+4. Now to check if everything works, type `./kci_rootfs --help` it should produce below help message.
+
+    ```
+    usage: kci_rootfs [-h] [--yaml-configs YAML_CONFIGS]
```

Comment:
There's a problem with extra blank lines being added, maybe that was a copy-paste artifact?
```
$ ./kci_rootfs --help
usage: kci_rootfs [-h] [--yaml-configs YAML_CONFIGS]
                  {validate,list_configs,list_variants,build,upload} ...

optional arguments:
  -h, --help            show this help message and exit
  --yaml-configs YAML_CONFIGS
                        Path to the YAML configs file

Commands:
  {validate,list_configs,list_variants,build,upload}
                        List of available commands
    validate            Validate the rootfs YAML configuration
    list_configs        List all rootfs config names
    list_variants       List all rootfs variants
    build               Build a rootfs image
    upload              Upload a rootfs image
```

Reply:
I guess so copy-pasting output, added new blanks lines.